dora/core/server/worker/src/main/java/alluxio/worker/AlluxioWorker.java

```diff
@@ -53,18 +53,6 @@ public static void main(String[] args) {
     }
 
     CommonUtils.PROCESS_TYPE.set(CommonUtils.ProcessType.WORKER);
-    MasterInquireClient masterInquireClient =
-        MasterInquireClient.Factory.create(Configuration.global(), ServerUserState.global());
-    try {
-      RetryUtils.retry("load cluster default configuration with master", () -> {
-        InetSocketAddress masterAddress = masterInquireClient.getPrimaryRpcAddress();
-        Configuration.loadClusterDefaults(masterAddress, Scope.WORKER);
-      }, RetryUtils.defaultWorkerMasterClientRetry());
-    } catch (IOException e) {
-      ProcessUtils.fatalError(LOG,
-          "Failed to load cluster default configuration for worker. Please make sure that Alluxio "
-              + "master is running: %s", e.toString());
-    }
```

Comment:
also remove similar code  in `AlluxioJobWorker.java`?

Reply:
and `alluxio.conf.Configuration#loadClusterDefaults` as well as their references?